### ISSUE TITLE
Whether it can be accelerated with GPU

### ISSUE THREAD

**Issue by 21lihonghai** [codeforreal1/compressO#18]
I don't really understand how this software works, but I see on the task manager that he is consuming a lot of cpu resources but the graphics card is not, I don't know if it's possible to make the software use the graphics card for acceleration and thus increase efficiency

**Comment by Sankyuubigan** [comment]
My cpu is running at 100% and my nvidia card 4070 ti super stands idle at 0%, i am waiting for slow video compression. too bad. too bad.

**Comment by antonsizikov** [comment]
Yeah its possible since this app uses FFmpeg to compress video. There is a command that show available hardware asseleration methods: `ffmpeg -hwaccels`. And than it can use it instead of default `libx264`. There are such methods for different platforms:
- `h264_nvenc` - Nvidia
- `h264_qsv` - Intel
- `h264_amf` - AMD
- `h264_videotoolbox` - Apple

In my experience it gives a gain in speed and efficiency, but the quality is often noticeably worse. So acceleration is a compromise, especially if you want the best possible quality.

**Comment by zgbjty** [comment]
> Yeah its possible since this app uses FFmpeg to compress video. There is a command that show available hardware asseleration methods: `ffmpeg -hwaccels`. And than it can use it instead of default `libx264`. There are such methods for different platforms:
> 
> * `h264_nvenc` - Nvidia
> * `h264_qsv` - Intel
> * `h264_amf` - AMD
> * `h264_videotoolbox` - Apple
> 
> In my experience it gives a gain in speed and efficiency, but the quality is often noticeably worse. So acceleration is a compromise, especially if you want the best possible quality.

I don't know about ffmpeg, hope to have more detailed configuration steps, please
